week-1/explore/objects.md

```diff
@@ -4,13 +4,25 @@ In this activity, we'll explore some additional concepts that you'll encounter i
 
 Open the Chrome devtools Console, type in `console.log` and then hit enter
 
+//////////////////////////////////////////////////////////////
 What output do you get?
+`log() { [native code] } ` , this is the output.
+Native code is binary data compiled to run on a processor, such as an Intel x86-class processor. The code is written in all 1s and 0s that must conform to the processor's instruction set architecture (ISA). Native code provides instructions to the processor that describe what tasks to carry out.[no idea what this means ].
 
+///////////////////////////////////////////////////////
 Now enter just `console` in the Console, what output do you get back?
+`console {debug: ƒ, error: ƒ, info: ƒ, log: ƒ, warn: ƒ, …}` this is the out put if we enter just console ,
+`debug` , `error` , and ... are console object methods . for example we use `console.log()` to log out general messages .
+//////////////////////////////////////////////////////////
 
-Try also entering `typeof console`
+Try also entering `typeof console`:
+it's a JS object.
 
 Answer the following questions:
 
 What does `console` store?
```

Comment:
This isn't the case - maybe do a little extra research ( don't spend a long time! ) on `console`.

Reply:
i will